File: src/main/java/com/damianryan/octopus/model/ElectricityDualRate.java
package com.damianryan.octopus.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AccessLevel;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.FieldDefaults;

@Data
@NoArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class ElectricityDualRate {

    @JsonProperty("electricity_day")
    int electricityDay;

    @JsonProperty("electricity_night")
    int electricityNight;
}

File: src/main/kotlin/com/damianryan/octopus/model/ElectricityDualRate.kt
package com.damianryan.octopus.model

import com.fasterxml.jackson.annotation.JsonProperty

data class ElectricityDualRate(
    @JsonProperty("electricity_day") val electricityDay: Int = 0,
    @JsonProperty("electricity_night") val electricityNight: Int = 0,
)